### ISSUE TITLE
Bug on compilation

### ISSUE THREAD

**Issue by alphaonex86** [hardkernel/linux#238]
include/linux/dma-contiguous.h: In function 'dma_get_cma_size_int_byte':
include/linux/dma-contiguous.h:147:70: error: expected ';' before '}' token
 unsigned long dma_get_cma_size_int_byte(struct device *dev) {return 0};
Fixed missing ";"

But now I have:
arch/arm64/mm/init.o: In function `dma_get_cma_size_int_byte':
init.c:(.text+0x30): multiple definition of`dma_get_cma_size_int_byte'
arch/arm64/mm/dma-mapping.o:dma-mapping.c:(.text+0x908): first defined here
scripts/Makefile.build:387: recipe for target 'arch/arm64/mm/built-in.o' failed

**Comment by mdrjr** [comment]
More details please.. 
Provided defconfig?
GCC ?

**Comment by alphaonex86** [comment]
Not the def config (I need some config into =y not =m):
http://pastebin.com/1gukuXDF
Gcc version 5.4.0

**Comment by f1vefour** [comment]
That's the issue, GCC 5.y

**Comment by alphaonex86** [comment]
Due to linux kernel or hardkernel patch?

**Comment by bkrepo** [comment]
No, not really. See f6a10cc commit. It is a ugly C syntax mistake.

**Comment by alphaonex86** [comment]
Same with gcc version 4.9.3, seam I have already this patch
